src/config.js

```diff
@@ -44,6 +44,10 @@ try {
 			type: 'array',
 			disableable: true
 		}),
+		PR_BODY: getInput({
+			key: 'PR_BODY',
+			default: [ '' ]
```

Comment:
Why make the default an array? Just use the empty string directly: `default: ''`

Reply:
Resolved in c5d2db7.